astro/src/content/articles/security/adaptive-mfa-user-experience.mdx

```diff
@@ -0,0 +1,104 @@
+
+Adaptive MFA is the logical evolution towards a more user-friendly approach. While traditional MFA simply requests an additional authentication after the password (one extra step for the end user), adaptive MFA gives administrators many possibilities for enforcing an environment and bolstering a company's security posture while keeping things simple for the (already busy) end user.
+
+This article explains how convenient adaptive MFA can be, how it improves authentication on products, apps, and environments, and why you need to consider implementing it.
+
+## Limitations of Traditional MFA
+
+A traditional approach to MFA is usually a huge step up in securing organizations. It solves [weak-password](https://www.keepersecurity.com/blog/2023/01/12/weak-vs-strong-passwords-how-to-spot-the-difference/) issues by adding an additional layer of authentication that makes [unauthorized access](https://security.tennessee.edu/unauthorized-access/) hard for most attackers.
+
+But like most things in life, MFA comes with certain drawbacks.
+
+### Users Find MFA Inconvenient
+
+Yes, cybersecurity experts and enthusiasts love this great cyberattacker stopper. But regular users find using MFA to be a real challenge.
+
+Most users are workers with a lot on their plate. They need to complete tasks, run off to meetings, answer emails, and so on, in a never-ending cycle. Having to take that extra step to log in and carry on with their work is a complete nuisance.
+
+Users' most common complaint about MFA is how time-consuming and "inconvenient" it is to authenticate twice. That second authentication layer often involves several steps for the user that require them to switch between tabs, apps, or devices. They might even need to wait for an SMS message that never reaches their phone.
+
+This hassle can lead to annoyed users who raise their voices. No matter how important IT and security departments know authentication is, decision-makers in management and on the board care more about productivity and user experience than "theoretical" security risks.
+
+### Not All MFA Is More Secure
+
+Another important thing to keep in mind is that some MFA authentication methods can ruin the very purpose of MFA itself by creating greater vulnerabilities.
+
+For example, SMS and email authentication are the least secure MFA authentication methodologies. Getting a six-digit code in an email account that's not secured by MFA and might have much less robust password protection can leave you exposed.
+
+Likewise, SMS MFA is vulnerable to hackers that perform [SIM swap](https://en.wikipedia.org/wiki/SIM_swap_scam) attacks. They can clone SIM cards to have messages and calls redirected to the attacker's phone.
+
+MFA for the sake of MFA is not the same as using MFA to strengthen security.
+
+## Adaptive MFA Is the Answer
```

Comment:
I might be reading the markdown wrong here but this is odd to have as a heading and then just have 1 sentence below it?

Reply:
I don't love how they did that. I see the idea. Give an intro, then go into H3s. I'm going to rework that somewhat.